src/client.ts

```diff
@@ -70,7 +70,11 @@ export default class UnleashClient extends EventEmitter {
     return enabled;
   }
 
-  isFeatureEnabled(feature: FeatureInterface, context: Context, fallback: Function): boolean {
+  isFeatureEnabled(
+    feature: FeatureInterface | undefined,
+    context: Context,
+    fallback: Function,
+  ): boolean {
```

Comment:
Why are we changing the accepted parameter value? Surely, there's no point in ever calling this with a feature that's `undefined`? Wouldn't it be better to only allow you to call it with a defined feature?

Reply:
yes! Very good point, this is just the result of type refactoring, but I can look into this as it definitely makes sense!